packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1149,14 +1176,6 @@ export default class SalesforceCRMService implements CRM {
     return null;
   }
 
-  private determinePersonRecordType(createEventOn: string): SalesforceRecordEnum {
```

Comment:
We're replacing this method with `determineRecordTypeById` since each record type has a set prefix we can check instead of relying on this.